app/queries/servers/thread.ts

```diff
@@ -52,8 +52,8 @@ export const observeThreadById = (database: Database, threadId: string) => {
     );
 };
 
-export const observeUnreadsAndMentionsInTeam = (database: Database, teamId: string) => {
-    return queryThreadsInTeam(database, teamId, true).observeWithColumns(['unread_replies', 'unread_mentions']).pipe(
+export const observeUnreadsAndMentionsInTeam = (database: Database, teamId?: string, includeDmGm?: boolean) => {
```

Comment:
I wonder if these should only return values when CRT is enabled, otherwise it could add mentions and cause at some point the user enabled the feature, then disabled it and some data still lives in the DB